docs/contracts.rst

```diff
@@ -733,6 +733,63 @@ For example:
        >>> rich_logs[0]['args']
        {'myArg': 12345}
 
+   The default ``WARN`` flag logs a warning to the console if the log has errors, and the log that raised the warning will be discarded. Any logs that didn't encounter errors during processing will be returned.
+   There are also a few other flags available:
+
+     - ``STRICT`` - stops all processing and raises the error encountered.
+     - ``IGNORE`` - returns any raw logs that raised an error with an added "errors" field, along with any other logs were able to be processed.
+     - ``DISCARD`` - silently discards any logs that have errors, and returns processed logs that don't have errors.
+     - ``WARN`` - logs a warning to the console for the log that has an error, and discards the log. Returns any logs that are able to be processed.
+
+   An ``EventLogErrorFlag`` needs to be imported from web3/event_log_error_flags.py.
```

Comment:
Maybe just make them importable from` web3.logs` and don't mention the class `EventLogErrorFlag` anywhere in the docs.